src/components/Tags/constants.ts

```diff
@@ -0,0 +1,4 @@
+export const DROP_INDICATOR = {
```

Comment:
I can't find an assignment of these consts to DropIndicator. It's possible to break the layout if we accidentally change Container's height to 30 for example

Reply:
Changed